opencga-app/app/scripts/azure/arm/mongodb/postdeploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#this script is wrapped into the mongodb template in base64
+#just here for reference
```

Comment:
Lets add a `set -e` so the script stops on the first error

Reply:
Added in https://github.com/opencb/opencga/pull/975/commits/1ec41d9d76237c8182b43e1b401c073730cfff12